ui/pages/confirm-approve/confirm-approve.stories-to-do.js

```diff
@@ -10,6 +10,7 @@ import { store, getNewState } from '../../../.storybook/preview';
 import { subjectMetadata } from '../../../.storybook/initial-states/approval-screens/token-approval';
 import ConfirmApprove from '.';
 
+// eslint-disable-next-line import/no-anonymous-default-export
 export default {
   title: 'Pages/ConfirmApprove',
   id: __filename,
```

Comment:
Had to add to fix linting issue because it no longer recognizes this as a story file